Videos/videos_info.xml

```diff
@@ -242,18 +242,53 @@
         <date>2023-06-14</date>
         <renesaslink>https://www.renesas.com/video/how-use-smart-configurator-walkthrough-settings-clocks-and-components</renesaslink>
     </info>
-    <info id="_Toc87439381" name="videoInfo" device="RX || RL78 || RH850 || COMMON" category="tutorials">
+    <info id="_Toc87439381" name="videoInfo" device="RX || RL78" category="features">

```

Comment:
Q: What is the new definition of a feature and a tutorial?

Reply:
I don't think there is any change to the definition of a feature and a tutorial. Previous categorization of videos was done purely by me using the naming of the videos as a gauge. For example, "How to..." videos are automatically tutorial videos.
Following the excel sheet, some "How to..." videos are re-categorized to features like the Blinky Project feature.